region/tourian/main.json

```diff
@@ -1266,34 +1265,36 @@
         },
         {
           "id": 3,
-          "name": "Mother Brain (Broken Glass)",
+          "name": "Mother Brain",
           "nodeType": "event",
           "nodeSubType": "boss",
           "locks": null,
-          "note": "Because there is no official event for finishing MB1, the requirements for getting from broken glass to MB2 will be in the 3-to-4 link rather than an unlock on this node"
+          "note": "The requirements for finishing MB1 is in the 3 to 4 link."
         },
         {
           "id": 4,
           "name": "Mother Brain (phases 2 and 3)",
           "nodeType": "event",
           "nodeSubType": "boss",
-          "yields": [
-            "DefeatMotherBrain",
```

Comment:
Oh, you're taking away the flag entirely? Why?

Reply:
Why have 2 events that trigger from the same thing? ZebesSetAblaze seem to be what all of the changes look for.

Reply:
I view it as not hurting anything vanilla while simplifying things for the combo rando, which has no guarantee of `ZebesSetAblaze` and has `DefeatMotherBrain` as part of its win conditions.

Reply:
Will reinsert DefeatMotherbrain after Discord discussion